ExTrip/Controllers/Room/RoomResultViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  RoomViewController.swift
+//  ExTrip
+//
+//  Created by Nguyễn Hữu Toàn on 15/02/2023.
+//
+
+import UIKit
+
+class RoomResultViewController: UIViewController {
+        
+    var bookingTime: HotelBookingModel?
+    
+    var rooms: [RoomModel]? {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    init(_ data: [RoomModel]?, time: HotelBookingModel) {
+        self.rooms = data
+        self.bookingTime = time
+        super.init(nibName: nil, bundle: nil)
+    }
```

Comment:
set `rooms = data` => reload tableView before init view => maybe cash of have some error